docs/userguide/gpu_and_tpu_usage.md

```diff
@@ -0,0 +1,166 @@
+This document was written for darts version 0.17.0.
+
+We assume that you already know about Torch Forecasting Models in Darts. If you're new to the topic we recommend you to read our [guide on Torch Forecasting Models](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html) first. Here is also an [Recurrent Neural Network (RNN) Model example](https://unit8co.github.io/darts/examples/04-RNN-examples.html), in which this guide is going to be based on.
+
+## Use CPU
+
+By default all models will run on CPU. As shown in the RNN example above, we'll import the Air Passenger dataset, as well as other necessary modules.
+```
+import numpy as np
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+from darts.dataprocessing.transformers import Scaler
+from darts.models import RNNModel
+from darts.metrics import mape
+from darts.datasets import AirPassengersDataset
+```
+
+Now we read and scale the data like this:
+
+```
+# Read data:
+series = AirPassengersDataset().load()
+series = series.astype(np.float32)
+
+# Create training and validation sets:
+train, val = series.split_after(pd.Timestamp("19590101"))
+
+# Normalize the time series (note: we avoid fitting the transformer on the validation set)
+transformer = Scaler()
+train_transformed = transformer.fit_transform(train)
+val_transformed = transformer.transform(val)
+series_transformed = transformer.transform(series)
+```
+
+Next we will create our RNN like this:
+```
+my_model = RNNModel(
+    model="RNN",
+    hidden_dim=20,
+    dropout=0,
+    batch_size=16,
+    n_epochs=300,
+    optimizer_kwargs={"lr": 1e-3},
+    model_name="Air_RNN",
+    log_tensorboard=True,
+    random_state=42,
+    training_length=20,
+    input_chunk_length=14,
+    force_reset=True,
+)
+```
+and fit it to the data:
+```
+my_model.fit(train_transformed, val_series=val_transformed)
+```
+where in the output we can see that no other processing unit is used to train our model:
+```
+GPU available: False, used: False
+TPU available: False, using: 0 TPU cores
+IPU available: False, using: 0 IPUs
+
+  | Name      | Type    | Params
+--------------------------------------
+0 | criterion | MSELoss | 0     
+1 | rnn       | RNN     | 460   
+2 | V         | Linear  | 21    
+--------------------------------------
+481       Trainable params
+0         Non-trainable params
+481       Total params
+0.004     Total estimated model params size (MB)
+
+Epoch 299: 100% 8/8 [00:00<00:00, 42.49it/s, loss=0.00285, v_num=logs]
+<darts.models.forecasting.rnn_model.RNNModel at 0x7fc75901cb10>
+```
+
+Now the model is ready to start predicting, which won't be shown here since it's included in the example linked in the start of this guide.
+
+## Use a GPU
+GPUs can dramatically improve the performance of your model in terms of processing time. We only need to instruct our model to use our machine's GPU through PyTorch Lighnting Trainer parameters, which are expressed as the `pl_trainer_kwargs` dictionary, like this:
```

Comment:
You could add a reference to the Pytorch Lightning doc (e.g. this page: https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#accelerator) here.

Reply:
@hrzn yes, argh I knew I forgot sth, thanks! OK, so now I pushed the update in my cloned repo (gsamaras/darts). What I should do now? It seems like that change is already reflected in this pull request?

Reply:
Thanks! Basically there's nothing more to do on your side for now. We'll wait for the checks to finish and then merge your branch. Thanks for the contribution :)